src/kirby/components/modal/modal.component.tns.scss

```diff
@@ -0,0 +1,28 @@
+@import "../../scss/native-imports";
+
+.modal-wrapper {
+  width: 100%;
+  height: 100%;
+  background-color: transparent;
+  background: transparent;
+  padding-top: 30;
+}
+
+.modal {
+  padding: 10;
```

Comment:
Would it be possible to use `padding-top: size('xs')` (to utilize the Kirby sizing functionality?)